Maths/EulersFunction.py

```diff
@@ -0,0 +1,9 @@
+def eulerPhi(n):
+    l = primeFactors(n)
+    l = set(l)
+    s = n
+    for x in l:
+        s *= (x - 1)/x 
+    return s  
+
+print(eulerPhi(100))
```

Comment:
You can define a "main" function whit a while(true) and use this function in this main function and get n from user.